django-stubs/contrib/syndication/views.pyi

```diff
@@ -11,17 +12,160 @@ def add_domain(domain: str, url: str, secure: bool = ...) -> str: ...
 
 class FeedDoesNotExist(ObjectDoesNotExist): ...
 
-class Feed:
-    feed_type: Any = ...
-    title_template: Any = ...
-    description_template: Any = ...
+Item = TypeVar("Item")
+Object = TypeVar("Object")
+
+class Feed(Generic[Item, Object]):
+    feed_type: Type[SyndicationFeed] = ...
```

Comment:
This should be set to a subclass (not instance) of `SyndicationFeed`, is this the right way to do it?

Reply:
Yes!